.github/workflows/go.yaml

```diff
@@ -7,6 +7,16 @@ on:
   pull_request:
 
 jobs:
+  # Go vulnerability check
+  vuln:
+    name: govulncheck
+    runs-on: ubuntu-latest
+    steps:
+      - name: govulncheck
+        uses: golang/govulncheck-action@v1.0.1
```

Comment:
If this is set to `@v1` does it:

1. work?
2. float to the latest `v1.x.x`?

If so, we should prefer that.

Reply:
Dependabot bumps these, which is why I like specifying fine-grained SemVer versions. Then we can see the diffs when they upgrade.